client/src/js/components/main/updatePasswordForm.jsx

```diff
@@ -105,7 +115,7 @@ class UpdatePassword extends React.Component {
                                     type="submit">Update my password
                                 </button>
                                 <p className="center">
-                                <a href="/login"><span>SIGN IN</span></a>
+                                <Link to="/login"><span>SIGN IN</span></Link>
                                 </p>
                             </div>
                         </div>
```

Comment:
@jattoabdul export default name should match base filename.
Source: https://github.com/airbnb/javascript#naming--filename-matches-export